src/js/index.js

```diff
@@ -77,13 +83,24 @@ function handleSort() {
 		.forEach(sortedChild => contentEl.append(sortedChild));
 }
 
+function handleExpired() {
+	if (showExpired.checked) {
+		search.set('expired', 'y');
+	} else {
+		search.delete('expired');
+	}
+}
+
 // The cascade is the function which handles calling filtering and sorting swag
 function cascade(force = false) {
 	force |= handleDifficulty(this === filterInput);
 	force |= handleTags(Boolean(this.el));
 	if (force || this === sortingInput) {
 		handleSort(this === sortingInput);
 	}
+	handleExpired();
```

Comment:
lint reports one missing new line before this line:
```
 src/js/index.js:101:2
  ✖  101:2  Expected blank line before this statement.  padding-line-between-statements
```

Reply:
For some reason I'm not seeing any of these linting errors when running `xo` locally 🤔

Reply:
Make sure you reinstall deps, the new version of XO added that rule

Reply:
Thanks, that did the trick for me 👍🏻